specs/market-api.yaml

```diff
@@ -1128,26 +1150,90 @@ components:
           type: string
           format: date-time
 
+    PropertyQueryEvent:
+      allOf:
+        - $ref: '#/components/schemas/Event'
+        - type: object
+          properties:
+            propertyQuery:
+              $ref: '#/components/schemas/PropertyQuery'
+
     ProposalEvent:
+      allOf:
+        - $ref: '#/components/schemas/Event'
+        - type: object
+      properties:
+        proposal:
+          $ref: '#/components/schemas/Proposal'
+
+    ProposalRejectedEvent:
       allOf:
         - $ref: '#/components/schemas/Event'
         - type: object
           properties:
-            proposal:
-              $ref: '#/components/schemas/Proposal'
+            proposalId:
+              type: string
+              readOnly: true
+            reason:
+              $ref: 'common.yaml#/components/schemas/Reason'
 
-    AgreementEvent:
+    AgreementProposalEvent:
```

Comment:
unused

Reply:
Used. It is a specialization of Event, therefore returned in Collect* webmethods, as agreed last week.

Reply:
You're right. Thx.
The description for CollectOffer/Demand webmethods does not mention it, and I was just searching for uses.
Description `CollectAgreementEvents` has full list of possible events. Pls consider supplementing also the descriptions for former ones.